setup.py

```diff
@@ -32,7 +32,8 @@
     install_requires=[
         "bravado-core >= 4.2.2",
         "crochet >= 1.4.0",
-        "fido >= 2.1.0, < 3.2.0",
+        "fido >= 2.1.0, <= 3.2.0",
```

Comment:
Well, are we expecting further major breakages after the rewrite? It's these (premature?) restrictions that cause upgrade troubles for larger projects. It's why we're stuck on an old requests version even though the newest one seems to work just fine. If you use tools that do proper restriction checking (like pip-faster does) then you won't be able to use a newer version. pip doesn't really do sophisticated checking.

I'd vote for removing this restriction if there is no reason for it right now and instead focus on providing a stable API with fido. That's what we should aim for, that's what we should expect. Let's not code for failure. And if we do need to add such a restriction it should be for a major version, i.e. fido < 4.